### ISSUE TITLE
Document Library - OLG Conference, Conference Manual issue

### ISSUE THREAD

**Issue by TomOwens5000** [josephr5000/confdb-public#166]
Issue we discussed about the Conference Manual creating an error message. Just tested it again and it does now come up as expected. This is working now. In the past it worked, then it did not and yielded an error message - and now it works again.
Did you, Joseph, fix something? Not sure why it is behaving differently from one time to another.

**Comment by josephr5000** [comment]
I don't recall anything specific, however on my recent DB upgrade I didn't migrate the files yet, as reported in #189.  If this answers your question, please close this issue.

**Comment by TomOwens5000** [comment]
Conference manual now works. Fixed.
Not closing because the US Manual Chapters 1, 2 and 3 do not work. See screen shot below.

![capture](https://cloud.githubusercontent.com/assets/20777996/19024987/bfcb3702-88c5-11e6-9aae-fb4cb0001209.PNG)
We have talked about this before. If I understand correctly you plan to somehow install (migrate?) them for me before we go live. Is that correct? If so and you have a reminder to do so we can close this. Let me know.
